.github/workflows/ci.yml

```diff
@@ -0,0 +1,134 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+      - /^\d+\.\d+\.\d(-alpha\.\d+|-beta\.\d+|-rc\.\d+)?$/
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-test:
+    name: Build and Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ["10", "12"]
+        os: [ubuntu-latest, macOS-latest]
+    env:
+      LOGS_DIR: /tmp/ngx-form-errors/logs
+      LOGS_FILE: /tmp/ngx-form-errors/logs/build-perf.log
+      TZ: "Europe/Brussels"
+      MAIN_NODE_VERSION: "10"
+      MAIN_OS: "ubuntu-latest"
```

Comment:
Mmm what if instead of defining these 2 variables here and check them in every step below... we could simply have 1 variable like `IS_RIGHT_PUBLISHING_ENVIRONMENT` that is TRUE only when we have the right version of node and the os platform... What do you think? The checks will be easier and simpler

Reply:
Yes, you're right! I try this 😊